lib/json_rpc_multiquery.dart

```diff
@@ -0,0 +1,80 @@
+// ignore_for_file: sort_constructors_first
+
+library json_rpc_multiquery;
```

Comment:
please use `web3dart` for the library name.

Reply:
Forget my previous comment, I believe this should a `part`.

Reply:
so.. i should put part web3dart